### PR TITLE
Fix edit section

### PR DESCRIPTION
**Pull request by victor-paumier**
- sort cities by name
- fix update logo file
- improve deployer: do not maintenance_down on deploy:failed

### CODE REVIEW

File: src/Entity/Section.php [hunk 1]
```diff
@@ -255,6 +255,10 @@ public function __construct()
     public function setLogoFile($logo): void
     {
         $this->logoFile = $logo;
+
+        if (null !== $logo) {
+            $this->updatedAt = new \DateTime();
```
Comment (JeremieSamson) [inline src/Entity/Section.php]: I don't get it, why do you need tochange the `updatedAt` value here ?
Isn't the `$logoName` changed when a new logo is uploaded ?

File: src/Form/SectionType.php [hunk 2]
```diff
@@ -38,6 +39,11 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'choice_label' => 'name',
                 'required' => true,
                 'label' => 'section.city',
+                'query_builder' => function (EntityRepository $er) {
+                    return $er->createQueryBuilder('c')
+                        ->orderBy('c.name', 'ASC')
+                        ;
```
Comment (JeremieSamson) [inline src/Form/SectionType.php]:
```suggestion
                    ;
```

File: src/Entity/Section.php (hunk 1 above)
Comment (victor-paumier) [reply src/Entity/Section.php]: https://github.com/dustin10/VichUploaderBundle/issues/8
Update only working if there is a change

Comment (JeremieSamson) [reply src/Entity/Section.php]: you don't reply my second question :)

Comment (victor-paumier) [reply src/Entity/Section.php]: $logoName will change when the $logoFile will be uploaded, $logoFile will only be uploaded if there is a change (updatedAt)

**Pull request merged**